### ISSUE TITLE
SAF Agent creation / No target found with name of created Agent

### ISSUE THREAD

**Issue by wszczurek** [oracle/weblogic-deploy-tooling#345]
WDT: 0.22
WDT Mode: OnLine
WebLogic: 10.3.6

Usecase:
- Create SAF Agent
- Create JMS Resource and target SAF Agent

The SAF Agent is created successfully.
See file model 0-WDTSAF-agent.yaml in attachment.

Once the Agent is created the JMS Resource creation fails.
See file model 1a-WDTSAF-jmsresource-sub-error-remote.yaml in attachement.

The Error received:
```
####<Apr 4, 2019 1:19:01 PM> <SEVERE> <update> <main> <WLSDPLY-09015> <updateDomain deployment failed: No target found with name WDTSAFAgent-1>

Issue Log for updateDomain version 0.22 running weblogic version 10.3.6.0 online mode:

SEVERE Messages:

        1. WLSDPLY-09015: updateDomain deployment failed: No target found with name WDTSAFAgent-1

Total:       WARNING :     0    SEVERE :     1

updateDomain.sh failed (exit code = 2)
```

Looks like the SAFAgent is not taken into account in __find_target method.
I have work-around this by changing the attribute_setter.py.
Will provide input via merge request what helped.
After this change i was able to execute models from both files.

**Comment by wszczurek** [comment]
The attachment for models.
[saf-agent-jmsres-model.zip](https://github.com/oracle/weblogic-deploy-tooling/files/3043675/saf-agent-jmsres-model.zip)

**Comment by rakillen** [comment]
Fixed with PR #346
